tests/models/edx/test_browser.py

```diff
@@ -9,7 +9,8 @@
 from pydantic.error_wrappers import ValidationError
 
 from ralph.exceptions import UnknownEventException
-from ralph.models.edx.browser import BaseBrowserEvent, PageClose
+from ralph.models.edx.browser import BaseBrowserEvent
+from ralph.models.edx.navigational import PageClose
```

Comment:
Ooops, it's something I forgot to move...

Reply:
I prefer `test_models_edx_ui_page_close_selector_with_valid_event` (only the event name) because with enrollment events for example, which names are very long we would exceed the column character limits and it would be difficult to read